### DISCUSSION TITLE
Why should I put my full name in my profile?

### DISCUSSION THREAD

**Discussion by ussego** [community/community#50524] [GitHub Education]
Why should I put my full name to be part of Github Education? I want that information to be private to the public.

**Comment by Neodevils** [comment]
You don't really have to put your full name on your profile. You also got an option for showing your nickname.

**Reply by ussego** [reply]
Where is the option? I can not find it.

**Reply by Neodevils** [reply]
> Where is the option? I can not find it.

![IMG_20230320_231621](https://user-images.githubusercontent.com/71941576/226455649-3160e02a-3f09-4c33-8e84-71d46ae84223.jpg)
When you tap to "Edit Profile" on your profile, you will see "Name" label.

**Reply by ussego** [reply]
![Screenshot_20230320-171900~2](https://user-images.githubusercontent.com/68031973/226456253-da2a846e-6b91-4686-9d42-1dbc41e72ddd.png)

**Reply by Neodevils** [reply]
You have to give a little bit information about yourself on your GitHub Profile.

**Reply by ussego** [reply]
> You have to give a little bit information about yourself on your GitHub Profile.

Does it have to be private information? If so, I won't. I have changed my Bio, do you think that it would be fine?

**Reply by ussego** [reply]
> You have to give a little bit information about yourself on your GitHub Profile.

Does it have to be private information? If so, I won't. I have changed my Bio, do you think that it would be fine?

**Reply by Neodevils** [reply]
@ussego is it fixed?

**Reply by ussego** [reply]
> @ussego is it fixed?

No but i dont care anymore

**Comment by 0xmycf** [comment]
Have you been able to verify your account? I've had the same issues. I've even gone that far as giving my full name and it was still rejecting me.
